src/Spectre.libClassifier.Tests/ObservationExtractorTest.cpp

```diff
@@ -18,6 +18,7 @@ limitations under the License.
 */
 
 #include <gtest/gtest.h>
+#include <memory>
```

Comment:
Simply a reference to memory STL solves ambiguity for `OutOfRangeException`? I thought this makes an opposite effect. 😨